Errors in hmda-reporting

Need to look into the root cause of these errors. 


```
│ [ERROR] [02/24/2020 20:06:47.996] [hmda-reporting-akka.actor.default-dispatcher-12] [akka.actor.ActorSystemImpl(hmda-reporting)] Error during processing of request: '20 (of class java.lang.Integer)'. Completing with 500 Internal Serve │
│ r Error response. To change default exception handling behavior, provide a custom ExceptionHandler.                                                                                                                                        │
│ scala.MatchError: 20 (of class java.lang.Integer)                                                                                                                                                                                          │
│     at hmda.reporting.api.http.ReportingHttpApi.$anonfun$filerListRoute$2(ReportingHttpApi.scala:41)                                                                                                                                       │
│     at akka.http.scaladsl.server.Directive$.$anonfun$addByNameNullaryApply$2(Directive.scala:145)                                                                                                                                          │
│     at akka.http.scaladsl.server.ConjunctionMagnet$$anon$2.$anonfun$apply$3(Directive.scala:213)                                                                                                                                           │
│     at akka.http.scaladsl.server.directives.BasicDirectives.$anonfun$mapRouteResult$2(BasicDirectives.scala:68)                                                                                                                            │
│     at akka.http.scaladsl.server.directives.BasicDirectives.$anonfun$textract$2(BasicDirectives.scala:161)                                                                                                                                 │
│     at akka.http.scaladsl.server.directives.BasicDirectives.$anonfun$mapRequestContext$2(BasicDirectives.scala:45)                                                                                                                         │
│     at akka.http.scaladsl.server.directives.BasicDirectives.$anonfun$textract$2(BasicDirectives.scala:161)                                                                                                                                 │
│     at akka.http.scaladsl.server.RouteConcatenation$RouteWithConcatenation.$anonfun$$tilde$1(RouteConcatenation.scala:44)                                                                                                                  │
│     at akka.http.scaladsl.server.directives.BasicDirectives.$anonfun$mapRouteResult$2(BasicDirectives.scala:68)                                                                                                                            │
│     at akka.http.scaladsl.server.directives.BasicDirectives.$anonfun$textract$2(BasicDirectives.scala:161)                                                                                                                                 │
│     at akka.http.scaladsl.server.directives.BasicDirectives.$anonfun$textract$2(BasicDirectives.scala:161)                                                                                                                                 │
│     at akka.http.scaladsl.server.directives.BasicDirectives.$anonfun$mapRouteResultWith$2(BasicDirectives.scala:74)                                                                                                                        │
│     at akka.http.scaladsl.server.directives.BasicDirectives.$anonfun$textract$2(BasicDirectives.scala:161)                                                                                                                                 │
│     at akka.http.scaladsl.server.RouteConcatenation$RouteWithConcatenation.$anonfun$$tilde$2(RouteConcatenation.scala:47)                                                                                                                  │
│     at akka.http.scaladsl.util.FastFuture$.strictTransform$1(FastFuture.scala:41)                                                                                                                                                          │
│     at akka.http.scaladsl.util.FastFuture$.transformWith$extension1(FastFuture.scala:45)                                                                                                                                                   │
└────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘

```

--- a/hmda-reporting/src/main/scala/hmda/reporting/api/http/ReportingHttpApi.scala
+++ b/hmda-reporting/src/main/scala/hmda/reporting/api/http/ReportingHttpApi.scala
@@ -71,7 +71,7 @@ trait ReportingHttpApi extends InstitutionComponent {
                     )
                     .toSet
               )
-//          case _ => Future(Set(HmdaFiler("","","")))
+          case _ => Future(Set(HmdaFiler("","","")))
         }
 
         onComplete(futFilerSet) {

--- a/hmda-reporting/src/main/scala/hmda/reporting/api/http/ReportingHttpApi.scala
+++ b/hmda-reporting/src/main/scala/hmda/reporting/api/http/ReportingHttpApi.scala
@@ -7,9 +7,9 @@ import akka.http.scaladsl.server.Route
 import akka.stream.ActorMaterializer
 import akka.util.Timeout
 import com.typesafe.config.ConfigFactory
-import ch.megard.akka.http.cors.scaladsl.CorsDirectives.{ cors, corsRejectionHandler }
+import ch.megard.akka.http.cors.scaladsl.CorsDirectives.{cors, corsRejectionHandler}
 import de.heikoseeberger.akkahttpcirce.FailFastCirceSupport._
-import hmda.model.institution.{ HmdaFiler, HmdaFilerResponse, MsaMd, MsaMdResponse }
+import hmda.model.institution.{HmdaFiler, HmdaFilerResponse, MsaMd, MsaMdResponse}
 import hmda.query.repository.ModifiedLarRepository
 import hmda.reporting.repository.InstitutionComponent
 import hmda.util.http.FilingResponseUtils.entityNotPresentResponse
@@ -17,8 +17,8 @@ import io.circe.generic.auto._
 import slick.basic.DatabaseConfig
 import slick.jdbc.JdbcProfile
 
-import scala.concurrent.ExecutionContext
-import scala.util.{ Failure, Success }
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.{Failure, Success}
 trait ReportingHttpApi extends InstitutionComponent {
 
   implicit val system: ActorSystem
@@ -38,7 +38,7 @@ trait ReportingHttpApi extends InstitutionComponent {
     path("filers" / IntNumber) { filingYear =>
       get {
 
-        val futFilerSet = filingYear match {
+        val futFilerSet: Future[Set[HmdaFiler]] = filingYear match {
           case 2018 =>
             institutionRepository2018
               .getFilteredFilers(bankFilterList)
@@ -71,6 +71,7 @@ trait ReportingHttpApi extends InstitutionComponent {
                     )
                     .toSet
               )
+//          case _ => Future(Set(HmdaFiler("","","")))
         }
 
         onComplete(futFilerSet) {

